scaffold/ddev/web-build/Dockerfile.drainpipe

```diff
@@ -0,0 +1,8 @@
+COPY install-task.sh /usr/local/bin
+RUN chmod +x /usr/local/bin/install-task.sh
+# Also update .github/workflows/ValidateTaskfile.yml if changing version.
+RUN install-task.sh -b /usr/local/bin -d v3.28.0
```

Comment:
This has been updated in main since, so we'll want to bump this before merging.

Reply:
Looks like I misread, and it hasn't been updated in main. I'll file a new issue.

https://github.com/Lullabot/drainpipe/blob/main/src/BinaryInstallerPlugin.php#L19-L22